db/blob/db_blob_basic_test.cc

```diff
@@ -757,7 +765,7 @@ TEST_F(DBBlobBasicTest, Properties) {
   oss << "Number of blob files: 2\nTotal size of blob files: "
       << total_expected_size
       << "\nTotal size of garbage in blob files: " << expected_garbage_size
-      << '\n';
+      << "\nBlob file space amplification: " << expected_space_amp << '\n';
```

Comment:
I'm surprised we assert the equality of text strings printed by RocksDB. :(

Reply:
Agree this is not ideal. On the other hand, at least we have a test case for the property